themes/digital.gov/src/scss/new/shortcodes/_do-dont-table.scss

```
@@ -0,0 +1,28 @@
+@use "uswds-core" as *;
+
+.dg-do-dont-table {
+  &__header--do {
+    @include u-border-top("05", "green-40v", "solid", "!important");
+    @include u-color("green-50v", "!important");
+  }
+
+  &__header--dont {
+    @include u-border-top("05", "red-50v", "solid", "!important");
+    @include u-color("red-50v", "!important");
+  }
+
+  &-row-do {
+    @include u-border-top("05", "green-40v", "solid");
+    @include u-color("green-50v");
+  }
+
+  svg {
+    @include u-margin-right(1);
+  }
+
+  &__body {
+    tr td {
+      width: 50%;
+    }
+  }
```

Consider simplifying the selector:

```suggestion
  &__body td {
      width: 50%;
  }
```